packages/obonode/obojobo-chunks-figure/adapter.js

```diff
@@ -21,6 +23,7 @@ const Adapter = {
 		model.setStateProp('width', null, p => parseInt(p, 10))
 		model.setStateProp('height', null, p => parseInt(p, 10))
 		model.setStateProp('alt', null)
+		model.setStateProp('captionWidth', ImageCaptionWidthTypes.IMAGE_WIDTH)
```

Comment:
This should lower-case the attribute and ensure that it only be one of the two acceptable values:

```suggestion
		model.setStateProp('captionWidth', ImageCaptionWidthTypes.IMAGE_WIDTH, p => p.toLowerCase(), [ImageCaptionWidthTypes.IMAGE_WIDTH, ImageCaptionWidthTypes.MAX_WIDTH])
```